### PR TITLE
Add UglifyJS block

### PR DESCRIPTION
**Pull request by sapegin**
Should I add it to `webpack-blocks` package too?

### CODE REVIEW

**Comment by jvanbruegge** [comment]
I think this block is not really needed, as it's basicly just
```js
addPlugins([
    new webpack.optimize.uglifyjs(options)
]);
```

**Comment by sapegin** [comment]
In this case you’ll have to copy paste options to each project, and defaults are far from ideal. Also it’s not `webpack.optimize.UglifyJsPlugin` but uglifyjs-webpack-plugin that supports ES6.

File: packages/uglify/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+```js
+{
+  plugins: [
+    new UglifyJSPlugin({
+      parallel: {
+        cache: true
+      },
+      uglifyOptions: {
+        ie8: false,
+        ecma: 5,
```
Comment (jvanbruegge) [inline packages/uglify/README.md]: If you want to support ES6, what does this do?

**Comment by jvanbruegge** [comment]
I see

File: packages/uglify/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@webpack-blocks/uglify",
+  "version": "1.0.0",
+  "description": "Webpack block for UglifyJS",
+  "main": "index",
+  "license": "MIT",
+  "author": "Andy Wermke <andy@dev.next-step-software.com>",
```
Comment (andywer) [inline packages/uglify/package.json]: Feel free to put your name in here 😉

**Comment by andywer** [comment]
Yes, that should then be part of the `webpack-blocks` package as well.

File: packages/uglify/package.json (hunk 2 above)
Comment (sapegin) [inline packages/uglify/package.json]: :shipit:

File: packages/uglify/README.md (hunk 1 above)
Comment (sapegin) [inline packages/uglify/README.md]: I read the docs again: we should leave the default values here: 8 for the parser and 5 for the printer.

Comment (sapegin) [inline packages/uglify/README.md]: And ie8 is false by default too ;-|

**Comment by sapegin** [comment]
@andywer All done! 🍕

**Comment by andywer** [comment]
Cool!

@sapegin No intention to add it to the `webpack-blocks` package immediately? 😉

**Comment by sapegin** [comment]
@andywer Done ;-)

File: packages/uglify/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+Based on [uglifyjs-webpack-plugin](https://github.com/webpack-contrib/uglifyjs-webpack-plugin) (not `webpack.optimize.UglifyJsPlugin`) which uses UglifyJS v3 (uglify-es) that supports ECMAScript 2015.
+
+
+## Usage
+
+```js
+const { createConfig, env, uglify } = require('webpack-blocks')
+
+module.exports = createConfig([
+  env('production', [
```
Comment (andywer) [inline packages/uglify/README.md]: 👍 for using `env('production', [])` in the sample

**Comment by andywer** [comment]
@andywer Memo to myself: Don't forget to publish.

**Pull request merged**